lib/sentry/integrations/oban/cron.ex

```diff
@@ -99,4 +99,12 @@ defmodule Sentry.Integrations.Oban.Cron do
     |> System.convert_time_unit(:native, :millisecond)
     |> Kernel./(1000)
   end
+
+  # MyApp.SomeWorker -> "my-app-some-worker"
+  defp slugify(worker_name) do
+    worker_name
+    |> String.split(".")
+    |> Enum.map(&(&1 |> Macro.underscore() |> String.replace("_", "-")))
+    |> Enum.join("-")
```

Comment:
```suggestion
    worker_name
    |> String.split(".")
    |> Enum.map_join("-", &(&1 |> Macro.underscore() |> String.replace("_", "-")))
    |> String.slice(0, 50)
```

But even this will force me to _edit_ every single Cron Monitor that I created, why? Because the Sentry form to add a monitor doesn't allow to define the monitor slug right away, so, if you want to adapt that slug, you have to edit. Also, I just tried and the web form only allow 50 characters as the maximum.

So, the slug by default is something like:

```elixir
"Core.MyDummyWorker" |> String.duplicate(5) |> String.downcase() |> String.replace(".", "") |> String.slice(0, 50)
```

Reply:
Yeah there's an issue here because [the Sentry check-in API](https://develop.sentry.dev/sdk/check-ins/) doesn't allow to programmatically change the *name*, only the monitor slug.

Great catch on the character limit though.